new/LIT6535DersYaqLenten.xml

```diff
@@ -52,6 +52,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     <text>
         <body>
             <div type="bibliography">
+              <listBibl type="editions" corresp="#t1syriac">
```

Comment:
I do not see the t1syriac title element above. where should the reference point to?
if the Syriac here is irrelevant, then just leave `#t1`, or omit the corresp completely
if this is a bibliography for the Syriac version of the text, you might need to add a t2syriac title for the Syriac version and then point to `#t2Syriac` here

Reply:
Thank you, I hope it's better now